### PR TITLE
Add man skill update

### PR DESCRIPTION
**Pull request by bobbyng626**
#33 Updates on this pull request, added counter after dead and more feature on 'Add Man' Skill for Rudolf.
1. 2 extra man will be create on create_man function
2. They shared weaker hp but same power with the original
3. The new man will blink after hp <= 0 and turns to invisible (However, it has not been removed from sp)
4. AI on Rudolf will still have to be developed for controlling the new man

### CODE REVIEW

**Comment by tyt2y3** [comment]
Can we create a new AI dedicated to Rudolf, and not preload all AI upfront?

**Comment by bobbyng626** [comment]
Add man skill bug: 
add_man_bug

new Rudolf attacks will 'hurt' another Rudolf

**Comment by tyt2y3** [comment]
Can we create a new AI file in LF2_19 dedicated to Rudolf?

File: LF/character.js [hunk 1]
```diff
@@ -937,6 +955,7 @@ define(['LF/livingobject', 'LF/global', 'core/combodec', 'core/util', 'LF/util']
             $.health.bdefend = 0
             if ($.health.hp <= 0) {
               $.die()
+              $.id_update('state14_die')
```
Comment (bobbyng626) [inline LF/character.js:958]: Check if I am NPC first

File: LF/character.js [hunk 2]
```diff
@@ -1795,6 +1818,23 @@ define(['LF/livingobject', 'LF/global', 'core/combodec', 'core/util', 'LF/util']
...
+            players.push({
+              name: '+man',
+              controller: { type: 'AIscript', id: 4 },
+              type: 'computer',
+              id: $.id,
+              team: $.team
+            });
+          }
+          if (players.length > 0) {
+            $.match.create_npc(players, {x: $.ps.x - 20, y: $.ps.y, z: $.ps.z}, 20, GC.default.health.mp_full);
```
Comment (bobbyng626) [inline LF/character.js:1834]: change the ps.x position

**Comment by bobbyng626** [comment]
New commit: added new Y velocity when falling onto ground regarding #38

File: LF/character.js [hunk 3]
```diff
@@ -1588,34 +1617,46 @@ define(['LF/livingobject', 'LF/global', 'core/combodec', 'core/util', 'LF/util']
         }
       }
 
-      $.injury(inj)
```
Comment (tyt2y3) [inline LF/character.js:1591]: is there a reason of moving this line?

Comment (bobbyng626) [reply LF/character.js:1591]: The offset_attack function will have return a delay if this line is NOT moved

Comment (tyt2y3) [reply LF/character.js:1591]: Thanks. Good to know

**Pull request merged**